interface/patient_file/history/encounters.php

```diff
@@ -41,6 +42,13 @@
     require_once("$srcdir/group.inc");
 }
 
+
+if($GLOBALS['enable_group_therapy'] && !acl_check("groups","glog",false, array('view','write'))){
```

Comment:
@oshriromatrix
This blocks only users with groups acl permission and not glog acl permission.
others who have no groups acl will get free pass.